Applications/ctkDICOMObjectViewer/ctkDICOMObjectViewerMain.cpp

```diff
@@ -50,7 +50,7 @@ int main(int argc, char** argv)
   else
     {
     s = QFileDialog::getOpenFileName( 0,
-     QFileDialog::tr("Choose an image file"), ".",
+     "Choose an image file", ".",
```

Comment:
Is this one removed because it is an application and not one that belong to the library ?

Reply:
Yes, it is just an application, which does not do any internationalization (does not load a dictionary, etc.), so it does not make sense to add `tr`. It would just pollute the translation file (translators would think they need to translate the string, but it is actually never used).